Scoring.py

```diff
@@ -0,0 +1,34 @@
+import random, pygame, sys, pygame.font
+from pygame.locals import *
+
+navyblue = (60,60,100)
```

Comment:
Avoid global variables when you can. It seems like you could make these class attributes.

Reply:
A special case of a variable that doesn't actually vary – that's used as a constant. Python-the-languge doesn't distinguish between constants and other variables, but Python-as-used-by-humans does. The [convention](https://www.python.org/dev/peps/pep-0008/#constants) for a constant is all uppercase `NAVY_BLUE`.